app/src/main/java/com/example/fittracker/dialog/MarvelDialog.java

```diff
@@ -35,26 +29,12 @@ protected void onCreate(Bundle savedInstanceState) {
         ButterKnife.bind(this);
         setTitle(null);
         setCancelable(false);
```

Comment:
can you put these two methods in the init method of the view?

Reply:
Done